src/app/pages/product-page/product-page.component.ts

```diff
@@ -1,21 +1,95 @@
 import { Component } from '@angular/core';
-import {NgClass} from '@angular/common';
+import {ActivatedRoute, Params, Router} from '@angular/router';
 import {FormsModule} from '@angular/forms';
+import {environment} from '../../../environments/environment';
+import {NgClass} from '@angular/common';
+import {HttpClient} from '@angular/common/http';
+import {Product} from './product-page.model';
+
 
 @Component({
   selector: 'app-product-page',
   standalone: true,
   imports: [
-    NgClass,
-    FormsModule
+
+    FormsModule,
+    NgClass
+
   ],
   templateUrl: './product-page.component.html',
   styleUrl: './product-page.component.css'
 })
+
+
+
 export class ProductPageComponent {
+  isNightMode: boolean = false;
```

Comment:
Az `isNightMode` már egyszer nem szerepel ezen a komponensen lentébb? nekem lokálban is errort dob erre.
![image](https://github.com/user-attachments/assets/4a8a8ddb-3f51-43a6-bc9b-d690cd031faa)